tools/integration_tests/local_file/helpers/helpers.go

```diff
@@ -0,0 +1,226 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helpers
+
+import (
+	"os"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/setup"
+)
+
+const (
+	FileName1                     = "foo1"
+	FileName2                     = "foo2"
+	FileName3                     = "foo3"
+	ImplicitDirName               = "implicit"
+	ImplicitFileName1             = "implicitFile1"
+	ExplicitDirName               = "explicit"
+	ExplicitFileName1             = "explicitFile1"
+	DirPerms          os.FileMode = 0755
+	FilePerms         os.FileMode = 0644
+	FileContents                  = "teststring"
+)
+
+func CreateLocalFile(fileName string, t *testing.T) (filePath string, f *os.File) {
+	// Creating a file shouldn't create file on GCS.
+	filePath = path.Join(setup.MntDir(), fileName)
+	f, err := os.OpenFile(filePath, os.O_RDWR|os.O_CREATE|os.O_TRUNC, FilePerms)
+	if err != nil {
+		t.Errorf("CreateLocalFile(%s): %v", fileName, err)
+	}
+
+	ValidateObjectNotFoundErr(fileName, t)
+	return
+}
+
+func ValidateObjectNotFoundErr(fileName string, t *testing.T) {
+	_, err := ReadObjectFromGCS(fileName)
+	if err == nil || !strings.Contains(err.Error(), "storage: object doesn't exist") {
+		t.Errorf("Incorrect error returned from GCS for file %s: %v", fileName, err)
+	}
+}
+
+func CloseLocalFile(f *os.File, fileName string, t *testing.T) {
```

Comment:
Is [this](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/util/operations/file_operations.go#L161) function useful instead of making separate function?

Reply:
This function will not fail the test if something goes wrong.